tasks/utilities/task_validate.wdl

```diff
@@ -54,256 +54,56 @@ task export_two_tsvs {
   }
 }
 
-task compare_two_tsvs {
+task theiavalidate {
   input {
-    String datatable1
     File datatable1_tsv
-    String datatable2
     File datatable2_tsv
-    File? validation_criteria_tsv
     String columns_to_compare
     String output_prefix
 
+    File? validation_criteria_tsv
+    File? column_translation_tsv
```

Comment:
Oh, great call in adding a `column_translation_tsv` input! Do you have an example of this option utilized on Terra?

Reply:
@kevinlibuit @sage-wright I did not test this option but I can do that now

Reply:
Tested: https://app.terra.bio/#workspaces/theiagen-validations/Theiagen_Scribner_Sandbox/job_history/45a03c16-8e84-46af-ab8b-759041b85426
Looks good though you can't have only one row in column_translation_tsv - will break the workflow. This is described in TheiaValidate repo